src/platformio.ini

```diff
@@ -105,3 +105,22 @@ upload_command =
 	python scripts/uart_upload.py "$SOURCE"
 
 # -----------------------------
+[env:BETAFPV_A03_develop]
+extends = env_gd32f130
+src_filter =
+	${env_gd32f130.src_filter}
+	+<targets/EWRF_E7082VM/*.c>
+build_flags =
+	${env_gd32f130.build_flags}
+	-D Pancake=1
```

Comment:
Pancake?

Reply:
Most likely it should be `A03` but I would change it to `BETAFPV_A03`.

Reply:
I missed it, change it to A03

Reply:
I have changed it to A03

Reply:
Why not to `BETAFPV_A03` as it is much more descriptive?

Reply:
Yep, BETAFPV_A03 will be better.  @aparknation can you please update.

I will go back and correct the other targets with manufacture_model.